bns-test/test/name_import.test.ts

```diff
@@ -0,0 +1,230 @@
+    "S02J6ZY48GV1EZ5V2V5RB9MP66SW86PYKKPVKG2CE",
+    "SZ2J6ZY48GV1EZ5V2V5RB9MP66SW86PYKKQ9H6DPR",
+    "SPMQEKN07D1VHAB8XQV835E3PTY3QWZRZ5H0DM36"
+  ];
+  const alice = addresses[0];
+  const bob = addresses[1];
+  const charlie = addresses[2];
+  const dave = addresses[3];
+
+  const cases = [{
+    namespace: "blockstack",
+    version: 1,
+    salt: "0000",
+    value: 96,
+    namespaceOwner: alice,
+    nameOwner: bob,
+    priceFunction: {
+      buckets: [7, 6, 5, 4, 3, 2, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+      base: 4,
+      coeff: 250,
+      noVowelDiscount: 4,
+      nonAlphaDiscount: 4,
+    },
+    renewalRule: 10,
+    nameImporter: bob,
+    zonefile: "0000",
+  }, {
+    namespace: "id",
+    version: 1,
+    salt: "0000",
+    value: 9600,
+    namespaceOwner: alice,
+    nameOwner: bob,
+    priceFunction: {
+      buckets: [6, 5, 4, 3, 2, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+      base: 4,
+      coeff: 250,
+      noVowelDiscount: 20,
+      nonAlphaDiscount: 20,
+    },
+    renewalRule: 52595,
+    nameImporter: alice,
+    zonefile: "1111",
+  }];
+
+  beforeEach(async () => {
+    const allocations = [
+      { principal: alice, amount: 10_000_000_000 },
+      { principal: bob, amount: 10_000_000 },
+      { principal: charlie, amount: 10_000_000 },
+      { principal: dave, amount: 10_000_000 },
+    ]
+    const binFile = getDefaultBinaryFilePath();
+    const dbFileName = getTempFilePath();
+    provider = await NativeClarityBinProvider.create(allocations, dbFileName, binFile);
+    bns = new BNSClient(provider);
+    await bns.deployContract();
+  });
+
+  describe("Given a launched namespace 'blockstack', owned by Alice, where Bob is nameImporter", () => {
+
+    beforeEach(async () => {
+      let receipt = await bns.namespacePreorder(cases[0].namespace, cases[0].salt, cases[0].value, { sender: cases[0].namespaceOwner });
+      expect(receipt.success).eq(true);
+      expect(receipt.result).include('Returned: u12');
+
+      receipt = await bns.namespaceReveal(
+        cases[0].namespace, 
+        cases[0].salt,
+        cases[0].priceFunction, 
+        cases[0].renewalRule, 
+        cases[0].nameImporter, { sender: cases[0].namespaceOwner });
+      expect(receipt.success).eq(true);
+      expect(receipt.result).include('Returned: true');
+      await bns.mineBlocks(1);  
+    });
+
+    it("Charlie trying to import 'alpha.blockstack' should fail", async () => {
+      var receipt = await bns.nameImport(cases[0].namespace, "alpha", charlie, cases[0].zonefile, { sender: charlie })
+      expect(receipt.success).eq(false);
+      expect(receipt.error).include('Aborted: 1011');
+    // });
+
+    // it("Bob trying to import 'alpha.blockstack' should succeed", async () => {
+      receipt = await bns.nameImport(cases[0].namespace, "alpha", bob, cases[0].zonefile, { sender: bob })
+      expect(receipt.success).eq(true);
+      expect(receipt.result).include('Returned: true');
+    // });
+
```

Comment:
Can you also add a test-case where Bob tries to import two names to the same owner, and checks that the second import fails because a name can have at most one non-expired non-revoked name?